kata-deploy/kata-deploy.yaml

```diff
@@ -0,0 +1,184 @@
+---
```

Comment:
This file appears to have a lot of boilerplate code. I'd much prefer we built the embedded `cri-o` and `cri-containerd` documents from a common source file (`kata-deploy.yaml.in` or similar) otherwise it's going to be very hard to maintain this.